api/client/client.go

```diff
@@ -26,8 +28,8 @@ type Client struct {
 
 // ListenEvent is used to send data or error values for Listen
 type ListenEvent struct {
-	data []byte
-	err  error
+	action es.Action
```

Comment:
Feeling a little friction using the `es.Action` in client code because of the `EntityID` alias for `string`.

Reply:
Fixed this by modeling a similar struct in the client package that just uses `string`.